[docs] adhoc credential options (#4561)
expo/expo: docs/pages/versions/unversioned/guides/adhoc-builds.md

![Invalid Profile](/static/images/adhoc-builds-invalid-profile.png)

# Frequently Asked Questions

## Distribution certificate CLI options

`expo-cli` will give you options to produce a distribution certificate, and this section describes these options in more detail.

### Creating a new certificate (recommended)

> This will create a new iOS distribution certificate with a password. This option won't be available if you've already reached Apple's limits of 2 active certificates in your development account.

Reviewer: ```suggestion
This will create a new iOS distribution certificate with a password. This option won't be available if you've already reached Apple's limit of two active certificates in your developer account.
```

### Revoking existing certificates

> Revoking an existing distribution certificate associated with an App is safe if:

Reviewer: ```suggestion
Revoking an existing distribution certificate associated with an app is safe if:
```

lowercase "app"

- Your App is distributed through the App Store and you are **NOT** on an Apple Enterprise account.

Revoking an existing distribution certificate associated with an App is **NOT** safe if:

- Your App is distributed through the App Store and you are on an Apple Enterprise account.

> - Your App is distributed Ad hoc (distributed outside of the App Store for testing purposes).

Reviewer: ```suggestion
- Your App is distributed ad hoc (distributed outside of the App Store for testing purposes).
```

> An App Store App gets re-signed with an Apple certificate when it goes on the store with non Enterprise accounts. Revoking the certificate therefore won't affect it. Enterprise Apps and Apps distributed Ad hoc use the original certificate, which means revoking it will cause the App to stop functioning on all devices it is installed on.

Reviewer: ```suggestion
An App Store App gets re-signed with an Apple certificate when it goes on the store with non-Enterprise accounts. Revoking the certificate therefore won't affect it. Enterprise Apps and Apps distributed ad hoc use the original certificate, which means revoking it will cause the App to stop functioning on all devices it is installed on.
```

### Uploading existing certificate

> If you have a password protected certificate, you can provide the path to your `.p12` file as well as your password for upload. We recommend following [this excellent guide on making a P12 file](https://calvium.com/how-to-make-a-p12-file/) for making your own certificates.

Reviewer: ```suggestion
If you have a password-protected certificate, you can provide the path to your `.p12` file as well as your password for upload. We recommend following [this excellent guide on making a P12 file](https://calvium.com/how-to-make-a-p12-file/) for making your own certificates.
```

> **Note:** this guide recommends leaving the P12's password blank, but a P12 password is required to upload your own certificate to Expo's service. Please enter a password when prompted.

Reviewer: ```suggestion
**Note:** this guide recommends leaving the P12's password blank, but a **P12 password is required** to upload your own certificate to Expo's service. Please enter a password when prompted.
```

## Push key CLI options

`expo-cli` will give you options to produce a push key, and this section describes these options in more detail.

### Creating a new push key (recommended)

> This will create a new push key. This option won't be available if you've already reached Apple's limits of 2 active keys in your development account.

Reviewer: ```suggestion
This will create a new push key. This option won't be available if you've already reached Apple's limit of two active keys in your developer account.
```
